src/input.rs

```diff
@@ -135,17 +135,12 @@ impl<'a> Input<'a> {
     }
 
     pub fn is_stdin(&self) -> bool {
-        if let InputKind::StdIn = self.kind {
-            true
-        } else {
-            false
-        }
+        matches!(self.kind, InputKind::StdIn)
```

Comment:
This could be `self.kind == InputKind::StdIn`.

Reply:
Just tried. You can't do it because InputKind doesn't implement `PartialEq`.
And you can't derive it because there is a `Box`.
![image](https://user-images.githubusercontent.com/11428655/98438374-f6803c00-20e9-11eb-8485-1a2de98bf306.png)

Maybe I am missing something..

Reply:
Oh, no. You are right. Thanks.